compiler/internal/codegen/codegen_config_marshalers.go

```diff
@@ -0,0 +1,447 @@
+package codegen
```

Comment:
I renamed this file to correct a typo in the filename (`marshallers` -> `marshalers`).

So I think diffing the two files would be best during this review - not sure why Github hasn't seen it's a rename